lib/middleware.js

```diff
@@ -1,35 +1,27 @@
-'use strict'
-
-const router = require('express').Router()
+const Router = require('router')
+const bodyParser = require('body-parser')
 const onFinished = require('on-finished')
 const onHeaders = require('on-headers')
 
 const { Logger } = require('./logger')
+const { LOG_LEVEL_NAMES } = require('./log-levels')
+
+const router = Router()
+
+router.use(bodyParser.json())
+router.use(bodyParser.urlencoded({ extended: true }))
 
 router.post('/log/:level', (req, res) => {
   req.logger = new Logger()
   const level = req.params.level.toLowerCase()
 
   try {
-    switch (level) {
-      case 'debug':
-        req.logger.debug(req.body.message)
-        break
-      case 'info':
-        req.logger.info(req.body.message)
-        break
-      case 'warn':
-        req.logger.warn(req.body.message)
-        break
-      case 'error':
-        req.logger.error(req.body.message)
-        break
-      case 'fatal':
-        req.logger.fatal(req.body.message)
-        break
-      default:
-        req.logger.info(req.body.message)
+    if (LOG_LEVEL_NAMES.includes(level)) {

```

Comment:
Since when we control the log level with a post request?

Reply:
We... dont? This is an endpoint for logging from the client. If you need to write a log from outside the module server, you could send a request here and it wil be logged, without any complications.

Reply:
ok. I jump to conclusion without checking the function itself. my bad